astropy/table/table.py

```diff
@@ -1521,6 +1521,40 @@ def _repr_html_(self):
     def __repr__(self):
         return self._base_repr_(html=False, max_width=None)
 
+    def _repr_latex_ (self, latexdict={}):
+        """
+        Format Table object into the LaTeX format and print to the screen.
+
+        Parameters
+        ----------
+        latexdict : str or dict
+            Dictionary of LaTeX formatting options or either one of these three
+            strings corresponding to pre-defined options, ``AA``, ``doublelines``,
+            ``template``. Defaults to ``{}``.
+
+        Notes
+        -----
+        Uses ``latex.Latex.write()`` function from ``astropy.io.ascii`` to change
+        the table format to latex. The default LaTeX formatting options defined
+        in ``latex.Latex`` are assumed if no ``latexdict`` argument is passed.
+        See: https://docs.astropy.org/en/latest/api/astropy.io.ascii.Latex.html#astropy.io.ascii.Latex
+
+        A new key ``latexdict`` corresponding to ``Latex.latex`` dictionary of
```

Comment:
Can we change that to use the `meta['latexdict']` if it already exists? The reason for this is that the `_repr_*` functions are typically called automatically and not by hand, so that a use would not set the paramters to this function e.g. if you just show a Table in the Jupyter notebook, then Jupyter will call `tab._repr_html_()` and you'll see the result of that. If a user wants to print a latex table "by hand" then can already do that:
```
tab.write(sys.stdout, format='ascii.latex')
```